eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/steps/VariantsAnnotLoadBatchTest.java

```diff
@@ -0,0 +1,195 @@
+        JobTestUtils.cleanDBs(dbName);
+
+        // first do a mock of a "variants" collection, with just the _id
+        VariantAnnotationLineMapper lineMapper = new VariantAnnotationLineMapper();
+        List<VariantAnnotation> annotations = new ArrayList<>();
+        for (String annotLine : vepOutput.split("\n")) {
+            annotations.add(lineMapper.mapLine(annotLine, 0));
+        }
+        MongoClient mongoClient = new MongoClient();
+        DBCollection variants = mongoClient.getDB(dbName).getCollection("variants");
+        DBObjectToVariantAnnotationConverter converter = new DBObjectToVariantAnnotationConverter();
+        for (VariantAnnotation annotation : annotations) {
+            String id = buildStorageId(annotation.getChromosome(), annotation.getStart(),
+                    annotation.getReferenceAllele(), annotation.getAlternativeAllele());
+            try {
+                variants.insert(new BasicDBObject("_id", id));
+            } catch (MongoException.DuplicateKey e) {
+                ; // ignore, we are just scaffolding the variants collection
+            }
+        }
+
+        // now, load the annotation
+        MongoOperations mongoOperations = new MongoTemplate(new MongoClient(), dbName);
+        MongoItemWriter<VariantAnnotation> writer = variantsAnnotLoadBatch.initWriter("variants", mongoOperations);
+
+        writer.write(annotations);
+
+        // and finally check that documents in DB have annotation (only consequence type)
+        DBCursor cursor = variants.find();
+
+        int cnt=0;
+        int consequenceTypeCount = 0;
+        while (cursor.hasNext()) {
+            cnt++;
+            VariantAnnotation annot = converter.convertToDataModelType((DBObject)cursor.next().get("annot"));
+            assertTrue(annot.getConsequenceTypes() != null);
+            consequenceTypeCount += annot.getConsequenceTypes().size();
+        }
+        assertTrue(cnt>0);
+        assertEquals(annotations.size(), consequenceTypeCount);
+
+        JobTestUtils.cleanDBs(dbName);
+    }
+
+    private String buildStorageId(String chromosome, int start, String reference, String alternate) {
+        StringBuilder builder = new StringBuilder(chromosome);
+        builder.append("_");
+        builder.append(start);
+        builder.append("_");
+        if(!reference.equals("-")) {
+            if(reference.length() < 50) {
+                builder.append(reference);
+            } else {
+                builder.append(new String(CryptoUtils.encryptSha1(reference)));
+            }
+        }
+
+        builder.append("_");
+        if(!alternate.equals("-")) {
+            if(alternate.length() < 50) {
+                builder.append(alternate);
+            } else {
+                builder.append(new String(CryptoUtils.encryptSha1(alternate)));
+            }
+        }
+
+        return builder.toString();
+    }
+
+    private final String vepOutput = "" +
```

Comment:
In this case is it not easier to have this here? Easy to read and no need to open a new file to check the content